elyra/cli/pipeline_app.py

```diff
@@ -526,15 +538,29 @@ def describe(json_option, pipeline_path):
     describe_dict["type"] = primary_pipeline.type
     describe_dict["version"] = primary_pipeline.version
     describe_dict["nodes"] = len(primary_pipeline.nodes)
+    describe_dict["scripts"] = set()
+    describe_dict["notebooks"] = set()
     describe_dict["file_dependencies"] = set()
     describe_dict["component_dependencies"] = set()
+    describe_dict["mounted_volumes"] = set()
     for node in primary_pipeline.nodes:
         # collect information about file dependencies
         for dependency in node.get_component_parameter("dependencies", []):
             describe_dict["file_dependencies"].add(f"{dependency}")
         # collect information about component dependencies
         if node.component_source is not None:
             describe_dict["component_dependencies"].add(node.component_source)
+        # collect information of mounted volumes
+        for mounted_volume in node.get_component_parameter("mounted_volumes", []):
```

Comment:
Use the constant value for `mounted_volumes` as defined in https://github.com/elyra-ai/elyra/blob/master/elyra/pipeline/pipeline_constants.py#L20. (See other comment, which includes a clarification what these constants were introduced for)